README.md

```diff
@@ -79,7 +79,6 @@ STAGE=DEV
 [nextgen]
 aws_access_key_id=[YOUR_AWS_ACCESS_KEY]
 aws_secret_access_key=[YOUR_AWS_SECRET_ACCESS_KEY]
-region=eu-west-1
```

Comment:
Think this `region` needs to be part of `~/.aws/config` , could we add a paragraph for that?